BunqSdk/Security/SecurityUtils.cs

```diff
@@ -127,6 +133,20 @@ private static string GenerateRequestHeadersSortedString(HttpRequestMessage requ
             );
         }
 
+        private static string GetHeaderNameCorrectyCased(string headerName)
+        {
+            headerName = headerName.ToLower();
+            headerName = headerName.First().ToString().ToUpper() + headerName.Substring(1);
```

Comment:
https://msdn.microsoft.com/nl-nl/library/system.globalization.textinfo.totitlecase(v=vs.110).aspx ?

Reply:
what emile says ;-)

Reply:
Regex is already used in the other SDK's. So for consistency i would like to keep using regex.

Reply:
@OGKevin This is what I meant: https://dotnetfiddle.net/QdElDN

Reply:
Yes I understand. But for the sake of maintaining, I prefer consistency over all the SDK's. So lets keep the SDK's as consistent with each other as we can.